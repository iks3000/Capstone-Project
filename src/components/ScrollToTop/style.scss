@import '../../assets/styles/_variables.scss';

.scroll {
    position: fixed;
    bottom: 0;
    right: 30px;

    .scroll-to-top {
        width: 30px;
        height: 30px;
        display: flex;
        justify-content: center;
        align-items: center;
        background-color: $oxfordBlue;
        border-radius: 4px 4px 0 0;
        padding: 5px;
        cursor: pointer;
        box-sizing: border-box;
        transition: all 0.3s ease;
        color: $white;
        z-index: 10;

        &:hover {
            color: $green;
        }
    }
}