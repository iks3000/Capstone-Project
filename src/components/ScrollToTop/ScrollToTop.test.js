import React from "react";
import { shallow } from "enzyme";
import ScrollToTop from "./index";

it("should render ScrollToTop component", () => {
  expect(shallow(<ScrollToTop />)).toMatchSnapshot();
});

