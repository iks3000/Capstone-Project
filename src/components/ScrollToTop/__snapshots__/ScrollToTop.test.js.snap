// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ScrollToTop component 1`] = `
<div
  className="scroll"
/>
`;
